opentelemetry-sdk/src/resource/mod.rs

```diff
@@ -61,7 +63,29 @@ impl Default for Resource {
     }
 }
 
+impl From<ResourceBuilder> for Resource {
```

Comment:
I don't think we need to implement this conversion. I believe the intuitive thing to do is call `build()` on a builder type and not `into()`.

Reply:
removed